diff -ruN parmetis/metis/include/metis.h parmetis_aster/metis/include/metis.h
--- parmetis/metis/include/metis.h	2013-03-30 17:24:50.000000000 +0100
+++ parmetis_aster/metis/include/metis.h	2019-04-29 13:43:58.000000000 +0200
@@ -30,7 +30,11 @@
  GCC does provides these definitions in stdint.h, but it may require some
  modifications on other architectures.
 --------------------------------------------------------------------------*/
+#ifdef INTSIZE32
 #define IDXTYPEWIDTH 32
+#else
+#define IDXTYPEWIDTH 64
+#endif


 /*--------------------------------------------------------------------------
